tests/suites/test_suite_md.function

```diff
@@ -338,3 +344,48 @@ void mbedtls_md_file(int md_type, char *filename,
     ASSERT_COMPARE(output, mbedtls_md_get_size(md_info), hash->x, hash->len);
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void md_psa_dynamic_dispatch(int md_type, int pre_psa_ret, int post_psa_engine)
```

Comment:
Good point! I agree that right now those two cases are never executed, as we have only one component where SHA256 & SHA224 are driver only, which is `test_psa_crypto_config_accel_hash_use_psa` which currently doesn't have MD-light enabled, so is skipping this entire test suite.

However, that's temporary and the next PR (#7271) will have MD-light enabled in that component. Also, since things are similar for all hashes, and the test cases for other hashes are executed, I'd say the test gap, while real, is not very worrying. So, would you agree it's acceptable to merge this PR with this small test gap considering it's going to be fixed very soon? Otherwise I can add a component for that (like I did with `accel_hash_keep_builtins`). Wdyt?

Reply:
Yes, I think it is harmless. This can be fixed in https://github.com/Mbed-TLS/mbedtls/pull/7271.